taskvine/test/TR_vine_python_serverless.sh

```diff
@@ -11,6 +11,9 @@ export PYTHONPATH=$(pwd)/../../test_support/python_modules/${CCTOOLS_PYTHON_TEST
 
 STATUS_FILE=vine.status
 PORT_FILE=vine.port
+TEST_FILE=vine_python_serverless.py
```

Comment:
I'd recommend against naming this variable `TEST_FILE`. It gives the impression that the file can be removed. Use PYTHON_SCRIPT?